docs/source/cloud_storage_advanced_usage.rst

```diff
@@ -26,14 +26,14 @@ to any of these methods.
         local_path=<path/to/file>,
         bucket_name=<bucket-name>,
         path_in_bucket=<path/to/file/in/bucket>,
-        metadata={"tags": ["blah", "glah", "jah"], "cleaned": True, "id": 3}
+        metadata={"id": 3, "labels": ["blah", "glah", "jah"], "cleaned": True, "colour": "blue"}
```

Comment:
isn't the id a uuid?

Reply:
It is for `Identifiables` but this is just metadata for any given file (this isn't a method of an `Identifiable`)